examples/static_react_task/run_task.py

```diff
@@ -85,8 +67,29 @@ def build_task():
     os.chdir(return_dir)
 
 
-build_task()
+@hydra.main(config_name='scriptconfig')
+def main(cfg: DictConfig) -> None:
+    task_dir = cfg.task_dir
+
+    def onboarding_always_valid(onboarding_data):
+        return True
+
+    shared_state = SharedStaticTaskState(
+        static_task_data=[
+            {"text": "This text is good text!"},
+            {"text": "This text is bad text!"},
+        ],
+        validate_onboarding=onboarding_always_valid,
+    )
+
+    build_task(task_dir)
+
+    db = get_db_from_config(cfg)
+    cfg = augment_config_from_db(db, cfg)
```

Comment:
I really wish we didn't have to do this augment step because the code would be so much more nicer, but i can't think of a solution lol.

Reply:
I _could_ have a util that takes in cfg, and return db and cfg?